core/src/test/java/org/infinispan/api/APINonTxTest.java

```diff
@@ -622,7 +665,7 @@ public void testMerge() throws Exception {
 
       // remove if null value after remapping
       cache.merge("A", "C", (oldValue, newValue) -> null);
-      assertEquals(null, cache.get("A"));
+      assertNull(cache.get("A"));
```

Comment:
I prefer `assertEquals(null, ` since the error message contains the value; `assertNull` just fails.

Reply:
ok